src/Microsoft.IdentityModel.Logging/NonPII.cs

```diff
@@ -26,10 +26,10 @@ public NonPII(object argument)
         /// <summary>
         /// Returns a string that represents the <see cref="Argument"/>.
         /// </summary>
-        /// <returns><c>Null</c> if the <see cref="Argument"/> is <c>null</c>, otherwise calls <see cref="System.ValueType.ToString()"/> method of the <see cref="Argument"/>.</returns>
+        /// <returns><c>Null</c> if the <see cref="Argument"/> is <see langword="null"/>, otherwise calls <see cref="System.ValueType.ToString()"/> method of the <see cref="Argument"/>.</returns>
         public override string ToString()
         {
-            return Argument?.ToString() ?? "Null";
+            return Argument?.ToString() ?? "null";
```

Comment:
Isn't it going to mess-up existing logs?

Reply:
+1 no need to make this change from Null => null.

Reply:
Changed to "Null"